pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
+		span, ctx = ot.StartSpanFromContext(ctx, fmt.Sprintf("%s-%d", methodName, i))
+		span.LogFields(log.Object("gRPC request", request))
```

Comment:
My long term plan is that logging the request/response is controlled by a user option (otgrpc exposes this, but I haven't come up with a clean way to integrate it into our streaming code). I agree we probably don't want to be logging req/resp in general.